2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_setup.py:_flush():76] Configure stats pid to 3922
2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_setup.py:_flush():76] Loading settings from /home/an920107/.config/wandb/settings
2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_setup.py:_flush():76] Loading settings from /home/an920107/Documents/Programming/C++/演算法/hw1/wandb/settings
2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'C++/演算法/hw1/testing.py', 'program': '/home/an920107/Documents/Programming/C++/演算法/hw1/testing.py'}
2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_init.py:_log_setup():506] Logging user logs to /home/an920107/Documents/Programming/C++/演算法/hw1/wandb/run-20230405_184122-gitma46d/logs/debug.log
2023-04-05 18:41:22,814 INFO    MainThread:3922 [wandb_init.py:_log_setup():507] Logging internal logs to /home/an920107/Documents/Programming/C++/演算法/hw1/wandb/run-20230405_184122-gitma46d/logs/debug-internal.log
2023-04-05 18:41:22,815 INFO    MainThread:3922 [wandb_init.py:init():546] calling init triggers
2023-04-05 18:41:22,815 INFO    MainThread:3922 [wandb_init.py:init():552] wandb.init called with sweep_config: {}
config: {}
2023-04-05 18:41:22,815 INFO    MainThread:3922 [wandb_init.py:init():602] starting backend
2023-04-05 18:41:22,815 INFO    MainThread:3922 [wandb_init.py:init():606] setting up manager
2023-04-05 18:41:22,817 INFO    MainThread:3922 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-04-05 18:41:22,820 INFO    MainThread:3922 [wandb_init.py:init():613] backend started and connected
2023-04-05 18:41:22,822 INFO    MainThread:3922 [wandb_init.py:init():701] updated telemetry
2023-04-05 18:41:22,830 INFO    MainThread:3922 [wandb_init.py:init():741] communicating run to backend with 60.0 second timeout
2023-04-05 18:41:23,315 WARNING MainThread:3922 [wandb_init.py:init():1165] interrupted
Traceback (most recent call last):
  File "/home/an920107/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1144, in init
    run = wi.init()
  File "/home/an920107/.local/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 744, in init
    result = run_init_handle.wait(
  File "/home/an920107/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/home/an920107/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/home/an920107/.local/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/usr/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2023-04-05 18:41:25,812 WARNING MsgRouterThr:3922 [router.py:message_loop():77] message_loop has been closed
